### ISSUE TITLE
Design Intent - Accessibility Feedback - [VES Transition Experience, Discover Your Benefits, Results Page 2.0]

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#123872]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Consider:** When filters are applied, consider updating the filter heading/accordion with the number of filters applied. This helps users orient to the experience, and manage their filters. https://design.va.gov/storybook/?path=/story/components-va-search-filter--active-filters 
- [ ] **Consider:** For filters in the left nav, considering indicating the number of results for each filter (see Figure 1 below). This allows users to understand the results before they apply them, preventing filtering on filters that have zero results. 
- [ ] **Should:** For a keyboard only user (including screen reader users), when navigating a select, it triggers the `onchange` event creating a change of context failure. Sort by selects should be paired with a button, or use a different component, so that the page isn't updated automatically as a user navigates. This can be disorienting for users. 

Figure 1 (number of results per filter)

<img width="2000" height="1587" alt="filter number" src="https://github.com/user-attachments/assets/69950907-3fda-4fad-a62e-043bdde9fff7" />

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app

**Comment by ajakabcin** [comment]
Closing this out - feature was implemented following design system guidance as discussed in the linked slack thread above.
